addons/event_crm/models/event_registration.py

```diff
@@ -207,7 +207,7 @@ def _get_lead_contact_values(self):
             if not valid_partner.email:
```

Comment:
Side-note: Commit messages are missing task-ids, but I'm guessing it's a work in progress.

Reply:
There is no task :D

Reply:
Updated.